queries.sql

```diff
@@ -1 +1,41 @@
 /* Find all animals whose name ends in "mon".*/
+SELECT *
+FROM animals
+WHERE name LIKE '%mon'
+
+/*List the name of all animals born between 2016 and 2019*/
+SELECT *
+FROM animals
+WHERE date_of_birth between '2016/01/01' and '2019/01/01'
```

Comment:
- [ ] The requirement for this query is to list the only name of animals. Please query the name from the table instead of the entire record.
   - You can consider this code `SELECT name FROM animals WHERE date_of_birth between '2016/01/01' and '2019/01/01';`

Reply:
Done